runtime/log.go

```diff
@@ -0,0 +1,26 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// Create a logger that can be passed around through the environment.
+// Loggers can be created based on the one returned from this method by calling
+// WithField or WithFields and specifying additional fields that the package
+// would like.
+func CreateLogger(level string) (*logrus.Logger, error) {
+	if level == "" {
+		level = "warn"
+	}
+
+	lvl, err := logrus.ParseLevel(level)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to parse logging level: %s\n", level)
```

Comment:
> For all errors that we wrap, we should include the underlying error in the message,

Unless we're explicitly handling the error... because we know what it means

but yeah, generally I agree...